.cursor-plugin/commands/octo-model-config.md

```
@@ -485,22 +492,18 @@ When invoked WITH arguments (e.g., `/octo:model-config codex gpt-5.6-sol`), skip
 
 2. **Set Model** (`<provider> <model>` or with `--session`):
    ```bash
-   # Read and validate
-   CONFIG_FILE="${HOME}/.claude-octopus/config/providers.json"
-   # Use jq to set the model
-   jq --arg p "<provider>" --arg m "<model>" '.providers[$p].default = $m' "$CONFIG_FILE" > "${CONFIG_FILE}.tmp.$$" && mv "${CONFIG_FILE}.tmp.$$" "$CONFIG_FILE"
+   scripts/helpers/octo-model-config.sh set "<provider>" "<model>" [--session]
    ```
 
    **Dot syntax** (`<provider>.<capability> <model>`):
    ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)